### ISSUE TITLE
Replace pystring.{h,cpp} by Boost functions

### ISSUE THREAD

**Issue by dictoon** [appleseedhq/appleseed-maya#72]
As far as I can tell, everything can be done using the facilities provided by `std::string` and/or `MString`.

**Comment by haggi** [comment]
Indeed, pystring uses std::string internally. It only simplfies some actions with a syntax known form python. Any special reason why you want to get rid of it (except the warnings it produces)?

**Comment by dictoon** [comment]
It just doesn't add much compared to `std::string` for our uses, it's also rather foreign to a C++ developer, and I'd rather have as little code as possible in the project :)

**Comment by dictoon** [comment]
It appears that the usage of pystring is more widespread than I initially thought, and some of the functions (such as `startsWith()`) are not available in `std` nor in `foundation`. So maybe it makes sense to keep it after all.

For now I'll close this issue...

**Comment by est77** [comment]
Maybe it could be replaced by boost string algorithms?

**Comment by dictoon** [comment]
Right, since we already have Boost that would be an option as well.
